# Request grasp computation

---
bool success
